package.json

```diff
@@ -15,7 +15,8 @@
     "@reduxjs/toolkit": "^1.7.0-rc.0",
     "@self.id/react": "^0.3.2",
     "@self.id/web": "^0.4.2",
-    "@xmtp/xmtp-js": "^4.4.1",
+    "@walletconnect/web3-provider": "^1.7.8",
```

Comment:
Where do you use this lib?
If you need the web3-provider, I wrote a function `getWeb3Provider()` in `/lib` which only uses ethers :)